icinga2x/Vagrantfile

```diff
@@ -65,6 +85,8 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
 
       # provider: virtualbox
       node_config.vm.provider :virtualbox do |vb, override|
+        node_config.ssh.private_key_path = "~/.ssh/id_rsa"
```

Comment:
Hm. I think that was a leftover from some playing around. 
I removed it from both openstack and virtualbox provider and the openstack provider still works ( I can ssh in )